roles/2-common/tasks/fl.yml

```diff
@@ -29,10 +31,20 @@
     - "{{ doc_root }}/common/images"
     - "{{ doc_root }}/common/assets"
     - "{{ doc_root }}/common/services"
-    - "{{ py3_dist_path }}/iiab"
 
-- name: Symlink from {{ doc_root }}/common/webfonts to {{ doc_root }}/common/fonts
+- name: File Layout - Symlink {{ doc_root }}/common/webfonts -> {{ doc_root }}/common/fonts
   file:
-    src: "{{ doc_root }}/common/fonts"
+    src: "{{ doc_root }}/common/fonts"    # /library/www/html
     path: "{{ doc_root }}/common/webfonts"
     state: link
+
+- name: File Layout - Create log file directories {{ apache_log_dir }} & {{ nginx_log_dir }}
```

Comment:
should be handled automatically when the deb package is installed.

Reply:
I'll comment out this stanza if you're sure?

Reply:
Do we still care about setting these 2 dirs' owner/group as {{ apache_user }} (generally both 'www-data') and permission to '0770' ?

(Or do we let Apache / NGINX set their own defaults going forward?)

Reply:
Indeed, it looks like NGINX's default ownership and permissions are somehow overwriting/overriding the actions of [fl.yml](https://github.com/iiab/iiab/blob/master/roles/2-common/tasks/fl.yml#L41-L50) here.

And consequently `/var/log/nginx` ownership/permissions seem to be is causing Captive Portal to fail (to create a log file there) at times, with more details here: #2168